flyteadmin/pkg/manager/impl/resources/resource_manager_test.go

```diff
@@ -54,6 +54,13 @@ func TestUpdateWorkflowAttributes(t *testing.T) {
 	_, err := manager.UpdateWorkflowAttributes(context.Background(), request)
 	assert.Nil(t, err)
 	assert.True(t, createOrUpdateCalled)
+
+	request = &admin.WorkflowAttributesUpdateRequest{
+		Attributes: &admin.WorkflowAttributes{},
+	}
+	_, failError := manager.UpdateWorkflowAttributes(context.Background(), request)
+	assert.Error(t, failError)
```

Comment:
Can we also make sure it returns the right error?

Reply:
and other similar places

Reply:
all new added error test cases have been error type and error message checked in the code